frontend/src/pages/MyPage.tsx

```diff
@@ -1,74 +1,62 @@
 import ListFilter from '@/components/ListFilter/ListFilter';
-import ProfileRunnerPostItem from '@/components/Profile/ProfileRunnerPostItem/ProfileRunnerPostItem';
+import TechLabel from '@/components/TechLabel/TechLabel';
 import Avatar from '@/components/common/Avatar/Avatar';
+import Button from '@/components/common/Button/Button';
 import { BATON_BASE_URL } from '@/constants';
 import { useToken } from '@/hooks/useToken';
 import Layout from '@/layout/Layout';
-import { GetRunnerProfileResponse } from '@/types/profile';
-import { ReviewStatus } from '@/types/runnerPost';
-import { SelectOption } from '@/types/select';
+import { GetMyPagePost, GetMyPageProfileResponse, MyPagePost } from '@/types/myPage';
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
-
-type ReviewPostOptions = SelectOption<ReviewStatus>[];
-
-const reviewPostOptions: ReviewPostOptions = [
-  {
-    value: 'NOT_STARTED',
-    label: '대기중인 리뷰',
-    selected: true,
-  },
-  {
-    value: 'IN_PROGRESS',
-    label: '진행중인 리뷰',
-    selected: false,
-  },
-  {
-    value: 'DONE',
-    label: '완료된 리뷰',
-    selected: false,
-  },
-];
+import githubIcon from '@/assets/github-icon.svg';
+import MyPagePostList from '@/components/MyPage/MyPagePostList/MyPagePostList';
+import { getRequest } from '@/api/fetch';
+import { PostOptions, runnerPostOptions, supportPostOptions } from '@/utils/postOption';
 
 const MyPage = () => {
-  const [runnerProfile, setRunnerProfile] = useState<GetRunnerProfileResponse | null>(null);
-
-  const [postOptions, setPostOptions] = useState<ReviewPostOptions>(reviewPostOptions);
-
+  const [myPageProfile, setMyPageProfile] = useState<GetMyPageProfileResponse | null>(null);
+  const [myPagePostList, setMyPagePostList] = useState<GetMyPagePost | null>(null);
+  const [postOptions, setPostOptions] = useState<PostOptions>(runnerPostOptions);
   const [isRunner, setIsRunner] = useState(true);
 
   const { getToken } = useToken();
 
-  const getRunnerProfile = async () => {
-    try {
-      const token = getToken()?.value;
-      if (!token) throw new Error('토큰이 존재하지 않습니다');
+  useEffect(() => {
+    if (isRunner) {
+      setPostOptions(runnerPostOptions);
+    } else setPostOptions(supportPostOptions);
```

Comment:
```suggestion
      setPostOptions(isRunner ? runnerPostOptions : supporterPostOptions);
```
삼항 연산자를 쓰면 좋을 것 같아요!
그리고 ruuner/supporter가 아니라 runner/support네요!